extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -309,7 +309,9 @@ export class AmpAd3PImpl extends AMP.BaseElement {
     }
     // If a fallback does not exist attempt to collapse the ad.
     if (!this.fallback_) {
-      this.attemptChangeHeight(0, () => this./*OK*/collapse());
+      this.attemptChangeHeight(0).then(() => {
+        this./*OK*/collapse();
```

Comment:
It'll defer to the next pass just like `#attemptChangeHeight`. Isn't that "vsync"?